packages/components/header/README.md

```diff
@@ -50,7 +50,7 @@ compiled JavaScript for all components `nhsuk.min.js` or the individual componen
               <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" height="34" width="34"><path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path></svg>
             </a>
           </li>
-          <li class="nhsuk-mobile-menu-container js-show">
+          <li class="nhsuk-mobile-menu-container">
```

Comment:
It seems like `js-show` is still referenced here https://github.com/nhsuk/nhsuk-frontend/blob/header-calculation/packages/components/header/_header.scss#L132